### PR TITLE
[skip ci] add doctest for Accuracy metric

### PR DESCRIPTION
**Pull request by sdesrozis**
Addresses #2265

Description:  add doctest for `Accuracy` metric

@ydcjeff 

Check list:

- [ ] New tests are added (if a new feature is added)
- [ ] New doc strings: description and/or example code are in RST format
- [ ] Documentation is updated (if required)

### CODE REVIEW

File: ignite/metrics/accuracy.py [hunk 1]
```diff
@@ -126,6 +114,93 @@ def thresholded_output_transform(output):
...
+
+            0.6666...
+
+        Multiclass case
+
+        .. testcode:: 2
+
+            metric = Accuracy()
+            metric.attach(default_evaluator, "accuracy")
+            y_true = torch.Tensor([2, 0, 2, 1, 0, 1]).long()
```
Comment (vfdev-5) [inline ignite/metrics/accuracy.py:141]: @sdesrozis tensors should be created as `torch.tensor(data)` and not `torch.Tensor(data)` (it was deprecated since v0.4.0). `torch.Tensor` is only used for matching object vs class, e.g. `isinstance(t, torch.Tensor)`.
Please fix that everywhere in doctest strings. Thanks
